datasets/diplomacy_detection/diplomacy_detection.py

```diff
@@ -0,0 +1,117 @@
+    booktitle = "Proceedings of the 58th Annual Meeting of the Association for Computational Linguistics",
+    month = jul,
+    year = "2020",
+    address = "Online",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/2020.acl-main.353",
+    doi = "10.18653/v1/2020.acl-main.353",
+    pages = "3811--3854",
+    abstract = "Trust is implicit in many online text conversations{---}striking up new friendships, or asking for tech support. But trust can be betrayed through deception. We study the language and dynamics of deception in the negotiation-based game Diplomacy, where seven players compete for world domination by forging and breaking alliances with each other. Our study with players from the Diplomacy community gathers 17,289 messages annotated by the sender for their intended truthfulness and by the receiver for their perceived truthfulness. Unlike existing datasets, this captures deception in long-lasting relationships, where the interlocutors strategically combine truth with lies to advance objectives. A model that uses power dynamics and conversational contexts can predict when a lie occurs nearly as well as human players.",
+}
+"""
+
+_DESCRIPTION = "The Diplomacy dataset contains pairwise conversations annotated by the sender and the receiver for deception (and conversely truthfulness).   The 17,289 messages are gathered from 12 games.  We train baseline and neural models to predict lies."
+
+_HOMEPAGE = "https://sites.google.com/view/qanta/projects/diplomacy"
+
+# _LICENSE = ""
+
+_TEST_DOWNLOAD_URL = "https://github.com/DenisPeskov/2020_acl_diplomacy/raw/master/data/test.jsonl"
+_TRAIN_DOWNLOAD_URL = "https://github.com/DenisPeskov/2020_acl_diplomacy/raw/master/data/train.jsonl"
+_VALIDATION_DOWNLOAD_URL = "https://github.com/DenisPeskov/2020_acl_diplomacy/raw/master/data/validation.jsonl"
+
+
+class DiplomacyDetection(datasets.GeneratorBasedBuilder):
+    """Diplomacy: A Dataset for Deception Detection. """
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "messages": datasets.Value("string"),
+                "sender_labels": datasets.ClassLabel(names=["deceptive", "truthful"]),
+                "receiver_labels": datasets.ClassLabel(names=["deceptive", "truthful", "no_annotation"]),
+                "speakers": datasets.Value("string"),
+                "receivers": datasets.Value("string"),
+                "absolute_message_index": datasets.Value("int64"),
+                "relative_message_index": datasets.Value("string"),
+                "seasons": datasets.Value("string"),
+                "years": datasets.Value("string"),
+                "game_score": datasets.Value("string"),
+                "game_score_delta": datasets.Value("string"),
+                "players": datasets.Value("string"),
+                "game_id": datasets.Value("string"),
+            }
+        )
```

Comment:
I just took a look at the data and it looks like every single field in the json are lists (except gameid).

Therefore for all the features above you should define them as `datasets.Sequence(..)`.

For example
```
"messages": datasets.Sequence(datasets.Value("string")).
```

Another example with ClassLabel:
```
"sender_labels": datasets.Sequence(datasets.ClassLabel(names=["deceptive", "truthful"])).
```

Can you please do this change ?

You'll need to regerenate the dataset_infos.json file after this change.